osquery/tables/system/darwin/efigy.cpp

```diff
@@ -0,0 +1,496 @@
+/*
+ *  Copyright (c) 2017-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#include <CoreServices/CoreServices.h>
+#include <IOKit/IOKitLib.h>
+
+#include <iostream>
+#include <sstream>
+#include <string>
+
+#include <boost/algorithm/string.hpp>
+#include <boost/optional/optional.hpp>
+
+#include <osquery/core.h>
+#include <osquery/logger.h>
+#include <osquery/sql.h>
+#include <osquery/tables.h>
+
+#include "osquery/core/conversions.h"
+#include "osquery/remote/http_client.h"
+#include "osquery/tables/system/hash.h"
+
+namespace pt = boost::property_tree;
+
+namespace {
```

Comment:
Can we use `namespace osquery` so we don't have `osquery::` for all the internal datatypes?